### PR TITLE
Release

### PR DESCRIPTION
**Pull request by bguillou**

### CODE REVIEW

File: release/release [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  })
+  const Body = createBody({ header, body })
+
+  console.log('# Create path')
+  const Key = `raw/${PROJECT_ID}/${repositoryID}-${releaseID}.md`
+
+  const s3 = new aws.S3()
+
+  console.log('# Put file on Amazon S3')
+  s3.putObject({ Bucket, Key, Body }, resp => {
```
Comment (kesslerdev) [inline release/release]: `err`

File: release/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM node:10
+
+LABEL "com.github.actions.name"="Github Deployment"
+LABEL "com.github.actions.description"="Push release note to S3"
+LABEL "com.github.actions.icon"="bookmark"
+LABEL "com.github.actions.color"="green"
+
+RUN mkdir -p /app
+ADD . /app
```
Comment (kesslerdev) [inline release/Dockerfile]: better if copy only package.json & yarn lock, then run yarn
& copy sources after , in order to use the docker build cache
```Dockerfile
...
ADD package.json yarn.lock /app
RUN cd /app && yarn --pure-lockfile
...
```

File: release/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Could be tested by:
+- Define env variables
+- `yarn`
+- `yarn start`
+
+```
+action "Push release note to S3" {
+  uses = "inextensodigital/actions/release@master"
+  secrets = ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_PROFILE"]
```
Comment (shouze) [inline release/README.md]:
```suggestion
  secrets = ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"]
```

File: release/release [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    if (!resp) {
+      console.log('# File send on Amazon S3')
+      process.exit(0)
```
Comment (shouze) [inline release/release]: You can remove it, by default exit code is 0 on success

**Pull request merged**